src/Shouldly/ShouldlyMessage.cs

```diff
@@ -240,20 +240,21 @@ public string CreateExpectedErrorMessage()
         private static string CreateActualVsExpectedMessage(IShouldlyAssertionContext context)
         {
             var codePart = context.CodePart;
-            string message = string.Format(@"
-    {0}
-        {1}
-    {2}
-        but was
-    {3}",
-                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(), context.Actual.ToStringAwesomely());
+            var message = string.Format(@"
+            {0}
```

Comment:
Please fix the indenting of the string. This will put a heap of extra whitespace into the error messages.